docs/conf.py

```diff
@@ -81,6 +82,13 @@
 version = get_version("zarr")
 release = get_version("zarr")
 
+redirects = {
+     "spec": "https://zarr-specs.readthedocs.io",
```

Comment:
still not working for:
https://zarr--2555.org.readthedocs.build/en/2555/spec
https://zarr--2555.org.readthedocs.build/en/2555/spec/
https://zarr--2555.org.readthedocs.build/en/2555/spec/index.html
https://zarr--2555.org.readthedocs.build/en/2555/spec/v1/
https://zarr--2555.org.readthedocs.build/en/2555/spec/v1/index.html

Reply:
None of those are correct links for the current pages before this PR. The correct ones (that redirect for me) are:

- https://zarr--2555.org.readthedocs.build/en/2555/spec.html
- https://zarr--2555.org.readthedocs.build/en/2555/spec/v1.html
- https://zarr--2555.org.readthedocs.build/en/2555/spec/v2.html
- https://zarr--2555.org.readthedocs.build/en/2555/spec/v3.html

Reply:
well that would explain it 🤦🏽